.devcontainer/Dockerfile

```diff
@@ -17,3 +17,7 @@ RUN chown -R ${USERNAME}:${USERNAME} "/home/${USERNAME}"
 
 # log completion - make sure this is last
 RUN echo "$(date +'%Y-%m-%d %H:%M:%S')    docker build complete" >> "/home/${USERNAME}/status"
+#checkov:skip=CKV_DOCKER_2: No healthcheck is needed 
```

Comment:
Do the statements have to be at the end of the file? imo would be more organized if we can put this at the top